step-templates/azure-database-execute-sql-cmd.json

```diff
@@ -0,0 +1,135 @@
+        }
+      },
+      {
+        "Id": "3fe35f76-f140-48df-be0c-2a927069bc8b",
+        "Name": "azDbSqlCmd.dbName",
+        "Label": "Database Name",
+        "HelpText": "The name of the database to execute the sql command against",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "bbe9a203-89db-4286-a684-c95748543805",
+        "Name": "azDbSqlCmd.AuthType",
+        "Label": "SQL Server Authentication Type",
+        "HelpText": "The type of authentication to use when connecting.",
+        "DefaultValue": "sql",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Select",
+          "Octopus.SelectOptions": "sql|SQL Authentication\nadPwd|Active Directory Password\nad|Active Directory Integration"
+        }
+      },
+      {
+        "Id": "539885ef-8ff9-407e-8e33-3bf211d58df6",
+        "Name": "azDbSqlCmd.userId",
+        "Label": "User Id",
+        "HelpText": "The user id to use when authenticating with the sql server",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "60af2045-ec59-4f4f-baf4-79e0ae8bd60d",
+        "Name": "azDbSqlCmd.userPwd",
+        "Label": "User Password",
+        "HelpText": "Used with SQL or Active directory Password authentication.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Sensitive"
+        }
+      },
+      {
+        "Id": "e631537b-c109-4678-a7e5-bbc8166cca78",
+        "Name": "azDbSqlCmd.connectionTimeout",
+        "Label": "Connection Timeout",
+        "HelpText": "The timeout for the query measured in seconds between 0 and 65534",
+        "DefaultValue": "0",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "9f787938-012a-45e9-ae58-5a06b1f4f012",
+        "Name": "azDbSqlCmd.sqlCmd",
+        "Label": "SQL Command",
+        "HelpText": "The sql command to execute",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "MultiLineText"
+        }
+      },
+      {
+        "Id": "273c678e-008e-4598-978e-9d7abaadbf6c",
+        "Name": "azDbSqlCmd.retainResults",
+        "Label": "Retain Results",
+        "HelpText": "The results file is saved for future analysis and readability",
+        "DefaultValue": "false",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      },
+      {
+        "Id": "1eab9b67-1d5f-4886-9216-80602dbf6bad",
+        "Name": "azDbSqlCmd.saveDbInfo",
+        "Label": "Save Database Info",
+        "HelpText": "When enabled, the database and server name will be included at the bottom of the file along with sql script ran.",
+        "DefaultValue": "false",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
```

Comment:
I think we should remove this parameter.

Reply:
should we also remove adding this information at the bottom of the CSV, or just always include it?  I had added is so that the developers looking at the data can compare to the sql statement.  This way if they didn't get the expected results, they can check the sql statement to make sure it is correct, or that it is talking to the correct db.

Reply:
addressed in commit 4245341